src/hooks/useInitialValues.ts

```diff
@@ -77,10 +81,28 @@ export function useInitialValues(
             formFields,
             encounterContext,
           );
+
+          if (field.questionOptions.calculate?.calculateExpression) {
+            const expression = field.questionOptions.calculate.calculateExpression;
+            const node: FormNode = { value: field, type: 'field' };
+            const context = {
+              mode: encounterContext.sessionMode,
+              patient: encounterContext.patient,
+            };
+            if (field.questionOptions.calculate.calculateExpression.includes('resolve')) {
```

Comment:
Using "resolve(" would more specific. Can you add a comment as to why this is necessary?

```suggestion
            if (field.questionOptions.calculate.calculateExpression.includes('resolve(')) {
```